packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -275,7 +275,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
         // URIs use slashes as delimiters. Handles use URIs.
         // Thus having slashes in types almost guarantees trouble down the road!
-        assert(!this.id.includes("/"), 0x13a /* `Data store ID contains slash: ${id}` */);
+        if (this.id.includes("/")) {
+            throw new UsageError("Id for FluidDataStoreContext cannot have '/' characters");
```

Comment:
We should say Fluid data stores (not add context) as context is implementation detail. Users only know and care about data stores.